_data/changelogs/component-accordion.yml

```diff
@@ -2,6 +2,16 @@ title: Accordion
 type: component
 changelogURL:
 items:
+  - date: 2023-05-03
+    summary: Added two new settings to customize accordion background colors.
+    summaryAdditional: You can now customize accordion button and content background colors with `$theme-accordion-button-background-color` and `$theme-accordion-background-color`.
+    isBreaking: false
+    affectsJavascript: false
+    affectsMarkup: false
+    affectsStyles: true
```

Comment:
One last thing - I just noticed that we now have an `affectsSettings` option. What do you think about applying it to both changelog entries here?

Reply:
Added it in df8fb4b.